lib/levels.js

```diff
@@ -24,7 +24,11 @@ const levelMethods = {
     const stream = this[streamSym]
     logFatal.call(this, ...args)
     if (typeof stream.flushSync === 'function') {
-      stream.flushSync()
+      try {
+        stream.flushSync()
+      } catch (e) {
+        // do nothing
```

Comment:
I actually think we should do something. @mcollina @jsumners what's your thoughts

Reply:
If you can't write a log message that the user will _maybe_ see, what do you intend to do to alert them to that message not being written? From the limited understanding I have of the issue, at this point everything is fucked at this point so 🤷‍♂

Reply:
right but it's worth knowing about? this just swallows it

if we throw an error there will be an understanding that something is wrong

or we can even just `console.warn` if throwing seems too drastic

Reply:
Remove the `try/catch` and it will throw. Wasn't the point of this to remove the throw?

Reply:
yep. but the message thrown has clearly caused confusion, we could throw an error with more of an explanation

Reply:
Sure, sounds good to me.

Reply:
🤷‍♂ @mcollina

Reply:
I think swallowing the error is ok here. Essentially we can not write a log line that says we are dying by a fatal error. Notifying a developer is usually done via log files.. which we cannot use in this stance. I think this is correct.

Reply:
Compromise: update the useless “do nothing” comment with a link to the discussion.